elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -833,6 +834,10 @@ public PersistentResource getRelation(String relation, String id) {
             Object idVal = CoerceUtil.coerce(id, idType);
             String idField = dictionary.getIdFieldName(entityType);
 
+            if (dictionary.isMappedInterface(entityType) && idField == null) {
```

Comment:
This has some unfortunate consequences. Is the goal that polymorphic collections are read-only at the collection level? I am happy accepting that for now until we have use cases for more in the future, but want to ensure this is what we're going for.

Reply:
Kinda, the problem is adding filters to @ManyToAny queries in hibernate isn't supported.  The only way to support that as of now is to write your own HQL manually.

You can still create and update. But filtering is a no go.

Reply:
As a corollary, one could not access any individual element within that collection either (i.e. filter to a single element with id). I suspect this is alright and intended though, based on what you've said.

Reply:
That is correct.

Reply:
Can you provide details why this isn't supported - maybe link to hibernate docs or stack overflow?